### DISCUSSION TITLE
Easing PVP Entry for New/Ungeared Folks- S1G, S2H and more!

### DISCUSSION THREAD

**Discussion by ChaoticMe** [chromiecraft/chromiecraft#7656] [Suggestions]
Sergeant Fuzzbutts here,

WoW pre-Cata is arguably awful for gearing in PVP or PVE, with the current phase of TBC being the worst part. There's a small influx of growth happening right now, with a notable amount of PVP interest. Tragically, the current PvP system is stifling participation due to its HIGH barriers to entry and limited opportunities for progression. Changes are urgently needed, so here's 3 that have been popular in Discord's PVP channel and/or World:

- Remove Rating Requirements: Make all S3 gear available without ratings to lower barriers for all players.
- S2 Gear for Honor: Allow S2 gear to be purchased with honor as a catch-up option.
- S1 Gear for Gold: Make S1 gear purchasable with gold for easy, quick entry into PvP.

**_Why this matters?_**

No Teams Close to Weapon/Shoulder Ratings:
Even **2** months into S3, **top** rewards are unattainable even for **top players**.

Declining PvP Activity:
Participation has dropped significantly this season across the board. Even on CTA weekends, match numbers are dropping with the matches less often being full. 3s WSG is not new player friendly.

Impossible Catch-Up:
New players and alts are stuck at least two seasons behind, worsening the spiral. The current PVP gear available for gold is 3 tiers and a quality level behind the newest S3 gear pieces, which aggravates an already grindy process. There should be a limit to the suffering for gear.

These changes would boost activity, help new players catch up, and create a healthier PvP ecosystem. It creates a diverse playerbase and smooths PVE gearing so there's fewer folks hitting Mythic Gruul in quest greens. It's a bandaid fix, but hopefully one that can hold until the blood clots.

**Comment by tbcxd** [comment]
Hopefully these changes will go live asap.

**Comment by minthes** [comment]
related discussion: 
https://github.com/orgs/chromiecraft/discussions/7652
